src/components/DialogStates.vue

```diff
@@ -0,0 +1,138 @@
+<script lang="ts">
+import { defineComponent, ref, computed, Ref } from 'vue';
+
+// Component used to render dialog window which you can fill with custom content
```

Comment:
This multi-line comment (for multi-line commenting use /* */ please) should be as component doc header on the first line.

Reply:
Refactored into component docs.